Neos.Neos/Resources/Private/Translations/en/Main.xlf

```diff
@@ -836,6 +836,27 @@
 			<trans-unit id="login.loggedOut.body" xml:space="preserve">
 				<source>Successfully logged out</source>
 			</trans-unit>
+			<trans-unit id="success.impersonateUser" xml:space="preserve" approved="yes">
```

Comment:
i agree, but can also be changed later ...

Reply:
Indeed just copied the labels from the Unikka package...
Can rename them, also need to migrate the neos-ui part from the package the evening/night because without that, the content module will have no back button.